src/locale/locales/ko/messages.po

```diff
@@ -0,0 +1,2629 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2023-11-05 16:01-0800\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: ko\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
```

Comment:
Please add your name here and in `Language-Team` for credits!

Reply:
> Please add your name here and in `Language-Team` for credits!

Yes, It's done.